### PR TITLE
Add OrbitFakeClient

### PR DESCRIPTION
**Pull request by dpallotti**
#### Add OrbitFakeClient
This is a command line client used to trigger `OrbitService` to take a capture,
in order to perform performance testing of `OrbitService` on YHITI.

Bug: http://b/185987548

Test: Run `OrbitService` on a gamelet running Trata, run `OrbitFakeClient` with
different command line options and verify from the logs of `OrbitService` that
a capture with the corresponding options is taken.

#### Package OrbitFakeClient in Orbit.zip
So that it can be used in YHITI.

Bug: http://b/185987548

Test: Manually triggered
https://fusion2.corp.google.com/invocations/fee8652a-2220-42c2-988d-e6e87ea4c933
and verified the content of the final Orbit.zip.

### CODE REVIEW

File: src/FakeClient/FakeClientMain.cpp [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+int main(int argc, char* argv[]) {
+  absl::SetProgramUsageMessage("Orbit fake client for testing");
+  absl::ParseCommandLine(argc, argv);
+
+  FAIL_IF(absl::GetFlag(FLAGS_pid) == 0, "PID to capture not specified");
+  FAIL_IF(absl::GetFlag(FLAGS_duration) == 0, "Specified a zero-length duration");
+
+  int32_t process_id = absl::GetFlag(FLAGS_pid);
+  LOG("process_id=%d", process_id);
+  uint16_t samples_per_second = absl::GetFlag(FLAGS_sampling_rate);
```
Comment (dimitry-) [inline src/FakeClient/FakeClientMain.cpp:77]: nit: maybe rename flag to samples_per_second as well?

**Review by dimitry- — APPROVED** [review]

File: src/FakeClient/FakeClientMain.cpp (hunk 1 above)
Comment (dpallotti) [reply src/FakeClient/FakeClientMain.cpp:77]: The name of the variable is to match the name of the parameter, but I prefer sampling rate. That's also how this flag is called in other places.

**Review by beckerhe — APPROVED** [review]
Sorry for being late. LGTM.

**Comment by dpallotti** [comment]
> Sorry for being late.

Late?

**Pull request merged**